src/Cryptol/Eval/Value.hs

```diff
@@ -22,10 +22,19 @@
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE PatternSynonyms #-}
 
 module Cryptol.Eval.Value
   ( -- * GenericValue
-    GenValue(..), ConValue
+    GenValue 
+      (VRecord, VTuple
+      , VEnum, VBit
+      , VInteger, VRational
+      , VFloat, VWord
+      , VStream, VFun
+      , VPoly, VNumPoly)
+  , ConValue
+  , pattern VSeq
```

Comment:
I'd recommend writing the export list like this:

```suggestion
    GenValue 
      (VRecord, VTuple
      , VEnum, VBit
      , VInteger, VRational
      , VFloat, VWord
      , VStream, VFun
      , VPoly, VNumPoly
      , VSeq) -- pattern synonym
  , ConValue
```

That way, if you write `import Cryptol.Eval.Value (GenValue(..))` elsewhere, then the pattern synonym will automatically be in scope. This will also avoid needing to enable `LANGUAGE PatternSynonyms` in quite so many places.